flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -511,15 +511,15 @@ case class Join(
 }
 
 case class CatalogNode(
-    tableName: String,
-    rowType: RelDataType) extends LeafNode {
+    rowType: RelDataType,
+    tablePath: String*) extends LeafNode {
```

Comment:
use arrays instead of var-args in internal classes is a convention? I could do it in this way, however,  var-args is more convenient to use than arrays in some cases.